src/core/common/message.hpp

```diff
@@ -1250,6 +1250,18 @@ class MessagePool : public InstanceLocator
      */
     Message *New(uint8_t aType, uint16_t aReserveHeader, uint8_t aPriority = kDefaultMessagePriority);
 
+    /**
+     * This method is used to obtain a new message with specified settings.
+     *
+     * @param[in]  aType           The message type.
+     * @param[in]  aReserveHeader  The number of header bytes to reserve.
+     * @param[in]  aSettings       A pointer to the message settings or NULL to set default settings.
```

Comment:
Add text about what the default settings are.

Reply:
Will add text.